scripts/script_scghg.py

```diff
@@ -0,0 +1,128 @@
+from pathlib import Path
+from pyfiglet import Figlet
```

Comment:
So this script provides a way to run the scghgs in bulk, without interaction with the command line tool, correct? But is otherwise the same functionality? If so, great 👍

Reply:
Yes, this script allows the user to interact with the run parameters by modifying variables in a script rather than through the command line interface. Otherwise the functionality is the same and uses the same input config.